pkg/postgresql/postgresql.go

```diff
@@ -0,0 +1,39 @@
+package postgresql
+
+import (
+	"context"
+	"fmt"
+	"github.com/jackc/pgx/v4"
+	"github.com/pkg/errors"
+)
+
+type Storage struct {
+	conn *pgx.Conn
+}
+
+func (st Storage) CloseConnection() {
+	st.conn.Close(context.Background())
+}
+
+func NewConnection(host string,
+	port string,
+	database string,
+	username string,
+	password string) (Storage, error) {
+	dsn := fmt.Sprintf("postgresql://%s:%s@%s:%s/%s", username, password,
+		host, port, database)
+	storage, err := pgx.Connect(context.Background(), dsn)
+	if err != nil {
+		errors.Wrap(err, fmt.Sprintf("unable to connect to database: %v\n", err))
+	}
+	return Storage{conn: storage}, err
+}
+
+func InsertData(storage Storage, name string, age int) error {
```

Comment:
```suggestion
func (s *Storage)InsertData(name string, age int) error {
```
я вот так вчера пытался тебе предложить сделать